engine/src/main/java/org/hibernate/validator/cfg/context/PropertyTarget.java

```diff
@@ -30,6 +30,45 @@
 	 * @param type The access type (field/property).
 	 *
 	 * @return A creational context representing the selected property.
+	 *
+	 * @deprecated Since 6.1. Planned for removal. Use one of {@link PropertyTarget#field(String)},
+	 * {@link PropertyTarget#getter(String)} instead, based on a property kind.
 	 */
+	@Deprecated
 	PropertyConstraintMappingContext property(String property, ElementType type);
+
+	/**
+	 * Selects a filed to which the next operations shall apply.
+	 * <p>
+	 * Until this method is called constraints apply on class level. After calling this method constraints
+	 * apply on the specified field property.
+	 * </p>
```

Comment:
Don't ask me why but in the javadoc, we don't close the `<p>`, we only open a new one when needed.